src/admin-views/settings/tickets-commerce/stripe/connect/active/connection.php

```diff
@@ -28,10 +28,10 @@
 		<span class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-text-name">
 			<?php echo esc_html( $name ); ?>
 		</span>
-		<a
+			<a
 				href="<?php echo esc_url( $disconnect_url ); ?>"
 				class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-text-disconnect-link"
```

Comment:
Needs to be aligned with the `<span>` above it. The parameters were indented too far out.
```suggestion
		<a
			href="<?php echo esc_url( $disconnect_url ); ?>"
			class="tec-tickets__admin-settings-tickets-commerce-gateway-connected-text-disconnect-link"
```

Reply:
True!